js/scripts.js

```diff
@@ -1,17 +1,12 @@
-function changeColor(){
-    var hex_numbers = ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9",
```

Comment:
I just fixed the return line here. You can't split an array onto multiple lines.